openmetadata-ui/src/main/resources/ui/src/components/common/entityPageInfo/ManageButton/ManageButton.tsx

```diff
@@ -55,10 +56,10 @@ const ManageButton: FC<Props> = ({
                 setIsDelete(true);
                 setShowActions(false);
               }}>
-              <SVGIcons alt="Delete" icon={Icons.DELETE_GRADIANT} />
+              <SVGIcons alt="Delete" icon={Icons.DELETE} />
               <div className="tw-text-left" data-testid="delete-button">
                 <p className="tw-font-medium" data-testid="delete-button-title">
-                  Delete {entityType} {entityName}
+                  Delete {entityType}
```

Comment:
Can it be simply "Delete" since the next line has a detailed description?
cc @devyani-kaushik

Reply:
Yes, @vivekratnavel, Make sense, I will update that.

Reply:
Done.